config/defaults.mk

```diff
@@ -276,6 +277,19 @@ ifeq ($(PETSC_FOUND),YES)
       -L$(abspath $(PETSC_DIR))/lib -lpetsc $(PETSC_LIB)
 endif
 
+SLEPC_ARCH := arch-linux2-c-debug
+SLEPC_DIR := $(MFEM_DIR)/../slepc/$(SLEPC_ARCH)
+SLEPC_VARS := $(SLEPC_DIR)/lib/slepc/conf/slepc_variables
```

Comment:
I downloaded version 3.13.2 of SLEPC but it doesn't contain this file. It does have `slepcvariables` in this location.  However, this does not seem to work for me.  MFEM's `make config` works without any errors but I cannot build MFEM because my include line doesn't contain SLEPC's include paths.  I've been trying to resolve this but I haven't found the answer yet.  One thing I did find is that `slepcvariables` does not define `SLEPC_INCLUDE`.  Also, `slepcvariables` seems to be a very different type of file from `petscvariables` so perhaps I'm not referencing the correct file here.

Reply:
Okay, I can build if I replace this path with `$(MFEM_DIR)/../slepc/lib/slepc/conf/slepc_variables` e.g. without the `$(SLEPC_ARCH)`.

Reply:
Yes this difference between SLEPc and PETSc is strange. I was using my system provided SLEPc installation (which does not define `SLEPC_ARCH`), and I just tried as you did with a vanilla SLEPc installation and `SLEPC_ARCH` is not used either (it reuses `PETSC_ARCH`). I will remove the references to `SLEPC_ARCH`.

Reply:
@mlstowell please let me know if https://github.com/mfem/mfem/pull/1487/commits/c7fe398bd763a5e5b3458d6e39cd61f781382f21 fixes this.

Reply:
Yes, I think that fixed it.  Thanks!